server/routes/dsl_route.ts

```diff
@@ -150,7 +150,7 @@ export function registerDslRoute(router: IRouter) {
       } catch (error) {
         const end = performance.now();
         context.searchRelevance.metricsService.addMetric(
-          METRIC_NAME.RELEVANT_SEARCH,
+          METRIC_NAME.SERACH_RELEVANCE,
```

Comment:
Typo: SERACH -> SEARCH

Also, it looks like the constant wasn't renamed in `index.ts`.

Reply:
Fixed